esmvalcore/cmor/_fixes/cmip5/gfdl_cm2p1.py

```diff
@@ -56,6 +58,50 @@ def fix_data(self, cube):
         return cube
 
 
+class Sit(Fix):
+    """Fixes for sit"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes bad bounds
+
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        time = cube.coord('time')
+        if self._fix_required(time):
+            new_bounds = np.empty(time.bounds.shape, time.bounds.dtype)
+            for i, point in enumerate(time):
+                date = point.units.num2date(point.points[0])
+                start = cftime.DatetimeJulian(date.year, date.month, 1)
+                if date.month < 12:
+                    end = cftime.DatetimeJulian(date.year, date.month + 1, 1)
+                else:
+                    end = cftime.DatetimeJulian(date.year + 1, 1, 1)
+                new_bounds[i, 0] = point.units.date2num(start)
+                new_bounds[i, 1] = point.units.date2num(end)
+            time.bounds = new_bounds
```

Comment:
I personally don't care much for `for` loops, so would suggest something a bit more pythonic, maybe

```suggestion
times = time.units.num2date(time.points)
starts = map(lambda c: cftime.DatetimeJulian(c.year, c.month, 1), times)
ends = map(lambda c: (cftime.DatetimeJulian(c.year, c.month + 1, 1) 
                                      if c.month < 12
                                      else cftime.DatetimeJulian(c.year + 1, 1, 1)), times)
time.bounds = time.units.date2num(np.stack([starts, ends], -1))
```